vsphere/datadog_checks/vsphere/config.py

```diff
@@ -31,6 +31,7 @@ def __init__(self, instance, log):
         self.password = instance['password']
         self.ssl_verify = is_affirmative(instance.get('ssl_verify', True))
```

Comment:
Will the wrapper work if users configure `tls_verify` and `tls_ca_cert`?

Reply:
No